ls1intum/Artemis
# `General`: Add tooltip to due date of exercise course card

### Checklist
#### General


#### Client

### Motivation and Context


#5029 redesigned the exercise header and added a tooltip to the dates of the exercise, when hovering over them. But an other place, namely the course card was left out.

### Description

This PR adds a short tooltip to the course exercise card that displays the due date, when hovering over it.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Exercise with due date

1. Log in to Artemis
2. Navigate to the course
3. Hover over the due date in the exercise card
4. Make sure that you are redirected, even when clicking directly on the due date

### Review Progress






#### Code Review
#### Manual Tests

### Screenshots


<img width="1116" alt="Bildschirm­foto 2022-12-09 um 11 39 13" src="https://user-images.githubusercontent.com/38322605/206699074-e0c8d20d-e516-4356-97ce-973c712a915f.png">
